mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -865,4 +934,114 @@ class Completions(
             else compareByRelevance(o1, o2)
       end compare
 
+  object MatchExtractor:
+    def unapply(path: List[Tree]) =
+      path match
+        // foo mat@@
+        case (sel @ Select(qualifier, name)) :: _
+            if "match".startsWith(name.toString()) && text.charAt(
+              completionPos.start - 1
+            ) == ' ' =>
+          Some(qualifier)
+        // foo match @@
```

Comment:
What would be the expected behaviour ?

Reply:
If we would have cases it would fall into next case 
```scala
// foo ma@tch (no cases)
        case (m @ Match(
              _,
              CaseDef(Literal(Constant(null)), _, _) :: Nil,
            )) :: _ =>
```
Should we provide exhaustive match for match with cases? What would be the expected behaviour ?